src/libaktualizr/http/httpinterface.h

```diff
@@ -14,15 +14,16 @@
 using CurlHandler = std::shared_ptr<CURL>;
 
 struct HttpResponse {
+  HttpResponse() = default;
   HttpResponse(std::string body_in, const long http_status_code_in, CURLcode curl_code_in,  // NOLINT
                std::string error_message_in)
       : body(std::move(body_in)),
         http_status_code(http_status_code_in),
         curl_code(curl_code_in),
         error_message(std::move(error_message_in)) {}
   std::string body;
-  long http_status_code;  // NOLINT
-  CURLcode curl_code;
+  int64_t http_status_code{0};
```

Comment:
We were using `long`, because it is the type in the signature of `curl_easy_getinfo`.

Reply:
OK, makes sense to change it back then.